coffee/react-countdown-clock.coffee

```diff
@@ -23,13 +23,15 @@ module.exports = React.createClass
     seconds: @props.seconds
 
   componentWillReceiveProps: (props) ->
-    @setState seconds: props.seconds
-
-  componentWillMount: ->
     @_setScale()
```

Comment:
I notice the demo page now accelerates after each iteration.

In hindsight I'm not sure this works. `_setScale` needs to be called after the `setState seconds: props.seconds` has been called. Even if you move the ordering around, I think the asynchronous nature of setState still messes this up.